klarna-on-demand/src/test/java/com/klarna/ondemand/RegistrationActivityTest.java

```diff
@@ -40,12 +47,16 @@
 
     @Before
     public void beforeEach() {
-        PowerMockito.mockStatic(Context.class);
+        mockStatic(Context.class);
         when(Context.getApiKey()).thenReturn("test_skadoo");
 
+        mockStatic(UrlHelper.class);
+        when(UrlHelper.registrationUrl((android.content.Context)anyObject())).thenReturn("my_url");
+
         Intent intent = new Intent(Intent.ACTION_VIEW);
         registrationActivityController = Robolectric.buildActivity(RegistrationActivity.class).withIntent(intent).create();
```

Comment:
`Robolectric.buildActivity(RegistrationActivity.class).withIntent(intent).create();` -> `registrationActivityController = Robolectric.buildActivity(RegistrationActivity.class).create();`
and remove line 56